packages/calcite-design-tokens/support/token-transformer/styleDictionary/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { Matcher } from "style-dictionary/types/Matcher.js";
+
+import { CalledTransformerFunction, TransformerTypeUnion } from "./transformer/utils.js";
+import { CalledFormatterFunction } from "./formatter/utils.js";
+
+// Just a list of all the options. We can pick from them later.
```

Comment:
Can you add more context to this comment? If it's a follow-up, maybe an issue would be more appropriate?